apps/i18n/common/sk_sk.json

```diff
@@ -96,17 +96,21 @@
   "assignHiddenUnitConfirm": "Vyzerá to, že ste v minulosti sekcii \"{sectionName}\" skryli \"{assignmentName}\". Stále chcete použiť tento kurz a sprístupniť ho?",
   "assigned": "Priradené",
   "assignedTo": "Priradené k",
+  "assignedToWithColon": "Pridelené:",
   "assignToSection": "Priradené k sekcii",
   "assignUnit": "Priradiť lekciu",
   "assignedToNewerVersion": "Učiteľ ti priradil novšiu verziu tohoto kurzu.",
   "assignmentSelectorCourse": "Vybrať kurz",
   "assignmentSelectorVersion": "Verzia",
   "assignmentSelectorUnit": "Vyber aktuálny kurz",
+  "atHomeBannerBody1": "Podpora pre rodičov a učiteľov zatvorených skôl.",
+  "atHomeBannerBody2": "Pozrite si naše navrhované vzdelávacie zdroje. ",
   "atHomeBannerLinkText": "Začať",
   "audioInitializeError": "Tvoj počítač nie je nastavený pre záznam zvuku. Skontroluj, či máš mikrofón s povolením na prístup k audio záznamu na tejto webovej stránke.",
   "audioSaveError": "Pri ukladaní nahrávky sa vyskytla chyba. Skús nahrať svoj záznam znova. Ak tento problém pretrváva, obnov stránku a skús ešte raz.",
   "authorizeGoogleClassrooms": "Kliknite sem pre autorizáciu Google Classroom.",
   "authorizeGoogleClassroomsText": "Pred tým ako môže prebehnúť synchronizácia s Google Classroom, musíte udeliť oprávnenie Code.org na prístup k účtu Google Class.",
+  "authorName": "**Autor:** {meno}",
```

Comment:
fyi @castro-jorge that it appears a translator translated several variables for this file and language

Reply:
Thanks for pointing this out, it's a new member from the Accenture team. I am getting in touch with her now.